protected/humhub/modules/admin/models/forms/MailingSettingsForm.php

```diff
@@ -76,6 +79,16 @@ public function attributeLabels()
         ];
     }
 
+    /**
+     * @inheritdoc
+     */
+    public function attributeHints()
+    {
+        return [
+            'systemEmailReplyTo' => Yii::t('AdminModule.settings', 'Optional. If used, check your DKIM/SPF in the DNS records for this email address'),
```

Comment:
Here are a few ideas;

`Optional: If used, remember to check your DKIM/SPF in the DNS records for this email address.`
or
`Optional: If used, remember to check your DKIM/SPF for this email address in the DNS records.`

Reply:
@funkycram Do we really need to address a correctly configured email server here? Might be more something for the documentation, that just confuses regular users.

I would rather write something like:
`Optional. Default reply address for system emails like notifications.`

@Semir1212 Any text idea?